app/views/mailbot/email_template.erb

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <!-- CSS -->
+  <link rel="stylesheet" href="main.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400,500,600,700,800" rel="stylesheet">
+  <style>
+  html {
+    scroll-behavior: smooth;
+  }
+
+  body {
+      background: #fff;
+      box-shadow: 0 0 2px rgba(0, 0, 0, 0.06);
+      color: #000;
+      font-family: 'Montserrat', sans-serif;
+      font-size: 16px;
+      line-height: 1.5;
+      margin: 0 auto;
+  }
+
+  h1,
+  h3,
+  h4,
+  h5,
+  h6 {
+      font-weight: 400;
+      line-height: 1.3;
+  }
+  p {
+    color: #0B3559;
```

Comment:
Actually, one more thing... Can we make these be `@conference.color`, or whatever the appropriate column is for the user-defined color on the conference settings page?